bin/src/run/watch.ts

```diff
@@ -71,15 +72,14 @@ export default function watch(
 					code: 'UNKNOWN_OPTION'
 				});
 
-			if (
```

Comment:
I'm not sure what potential side-effects are for this removal.

Reply:
because `processConfigs` was pulled out of the `start` method, it no longer has access to the  `screenWriter` variable, especially since that also happens later on.
I may be able to store it in some other variable and then reset it to stderr in the start method when it gets called.

Reply:
Ah I see, if you can keep the existing behaviour it may be best not to remove something that seems so intentional, especially given this is untested code.

Reply:
Makes sense. Let me update the PR then.